### PR TITLE
Milestone 2: Components, Actions and Reducers

### PR DESCRIPTION
**Pull request by tiagoit**
- App, Book, BooksList, BooksForm components.
- Implement Redux actions and reducers.
- Initialize Redux store.

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <th>category</th>
+        </tr>
+      </thead>
+      <tbody>
+        { books.map((book) => <Book book={book} key={book.id} />)}
+      </tbody>
+    </table>
+  );
+};
+
```
Comment (shloch) [inline src/containers/BooksList.js:23]: I dont see you connecting to redux store to fetch books
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png

Comment (shloch) [reply src/containers/BooksList.js:23]: use `mapStateToProps` to fetch books from the `store`

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @ferreirati 
It's all good, great job.

## [CHANGE REQUIRED]
1 issue left below, please resolve this and file for last re-review 😊😊

File: src/containers/BooksList.js (hunk 1 above)
Comment (tiagoit) [reply src/containers/BooksList.js:23]: Thanks Louis, we fixed that.

**Review by jofer86 — APPROVED** [review]
## 👋 👋 👋  Hello Team!! I was around the neighborhood and decided to stop by to do the follow-up review of this milestone!!
- I can see you delivered the changes required by my fellow TSE!! I have nothing left to say but... Approved! 👍 👍 

--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86.**

**Pull request merged**